### PR TITLE
Faq adjustments

### PR DESCRIPTION
**Pull request by noirbizarre**
Some style and content adjustments on the FAQ, the term and the credits.

### CODE REVIEW

File: udata_gouvfr/templates/terms.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{% extends theme("base.html") %}
+
+{% set meta = {
+    'title': 'Crédits',
```
Comment (davidbgk) [inline udata_gouvfr/templates/terms.html]: Wrong meta?

File: udata_gouvfr/views.py [hunk 2]
```diff
@@ -76,26 +71,17 @@ def dataconnexions():
...
-def faq_citizen():
-    return theme.render('faq/citizen.html')
-
-
-@blueprint.route('/faq/producer/', endpoint='faq_producer')
-def faq_producer():
-    return theme.render('faq/producer.html')
+@blueprint.route('/faq/', defaults={'section': 'home'})
+@blueprint.route('/faq/<string:section>/')
+def faq(section):
```
Comment (davidbgk) [inline udata_gouvfr/views.py]: I initially avoided a dynamic view to restrict the number of pages ;)

Comment (noirbizarre) [inline udata_gouvfr/views.py]: I can revert this but I find it more logic to route with:
```python
url_for('gouvfr.faq')
url_for('gouvfr.faq', section='citizen')
```

What do you think about it ?

Comment (davidbgk) [inline udata_gouvfr/views.py]: I think your version is better, let's keep it.

**Comment by davidbgk** [comment]
r+

**Pull request merged**